fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -13,15 +13,34 @@
 
 
 class _Lagrangian:
```

Comment:
Whole class is prefixed with underscore, so none of this will appear in main documentation ;-)

Still useful to start getting these described, though.

Reply:
It's just not possible to understand this without documentation. Even with all of this it's not much better IMO